### ISSUE TITLE
Accept arsenal exports for roles/groups additional items import

### ISSUE THREAD

**Issue by SiegeToaster** [Soliders-in-Arms-Arma-3-Group/sia3f#52]
Currently, additional items only accepts array with format ``["class1", "class2", "class3"]``.

Example ACE arsenal export: ``[[],[],[],[],[],[],"H_Hat_Tinfoil_F","G_Aviator",[],["ItemMap","ItemGPS","ItemRadio","ItemCompass","ACE_Altimeter",""]]``

The vanilla arsenal exports to code so that may be too complex to be worth it, but the ACE arsenal export only needs to be rearranged.

**Comment by L-McKendrick** [comment]
```sqf
_items =  [[],[],[],[],[],[],"H_Hat_Tinfoil_F","G_Aviator",[],["ItemMap","ItemGPS","ItemRadio","ItemCompass","ACE_Altimeter",""]];
_items = flatten _items;
_items = _items arrayIntersect _items select {_x isEqualType "" && {_x != ""}};
```
From [ACE Wiki](https://ace3.acemod.org/wiki/framework/arsenal-framework.html#23-arsenal-only-with-items-from-default-loadouts-see-section-4)

**Comment by SiegeToaster** [comment]
Resolved by b08e0c1e91709af237cc30578a5e8fb4c9d52395 and 69b65d917f48f9520db38ca4a976cb943d4df651
